README.md

```diff
@@ -136,31 +136,31 @@ Boardman, Mark Mavrogordato, Manjit Dosanjh, Steven Hancock, Thomas Blumensath*
 
 ## Contributors
 
-<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
```

Comment:
I can't see what changed here, but this should be untouched as its bot generated.

Reply:
ok, thanks for the correction, I will review it to correct it

Reply:
I reverted the change to see if it appeared the same as it appears in the GitHub interface, and no, it seems that GitHub thinks I have modified the "untouchable" thing, however, locally only my typo change is there.

It seems to be a mistake, because the code remains intact.